### ISSUE TITLE
Declare the project dead

### ISSUE THREAD

**Issue by KyNorthstar** [SwiftForWindows/SwiftForWindows#76]
It's been over 1 year since the last commit. This project is dead and should be presented as such

**Comment by jefflafay** [comment]
I agree. I was looking for a swift on windows option and this project is the top result on google which is misleading.

**Comment by eliaskas** [comment]
I'm so sorry for that...

**Comment by jhoughjr** [comment]
does it have to be dead? What are the issues with Swift 5.x? I would think ABI stabillity would make this project easier.

**Comment by tinysun212** [comment]
As long as you are interested, this project will not die.

**Comment by jhoughjr** [comment]
Then I suppose my question is what is the status of utilizing swift 5.x? I doubt I have the skillset but I think I saw instructions for building swift with MS C++. Were I not swamped I'd experiment. If Swift can be brought to windows in a manner on par with other Langs, it will spread its wings.

**Comment by KyNorthstar** [comment]
@tinysun212 I am interested, but I don't have nearly enough time in my life to be a Swift for Windows maintainer. I assume that this project has not had any commits in a long time because so many of us are in the same position.

It's nice to keep the enthusiasm! And I have plenty to go around! But this project cannot be presented as a modern solution for using Swift on Windows, since it only supports an outdated version of Swift and nobody is willing to fix that.

**Comment by nolanthomas** [comment]
> Then I suppose my question is what is the status of utilizing swift 5.x? I doubt I have the skillset but I think I saw instructions for building swift with MS C++. Were I not swamped I'd experiment. If Swift can be brought to windows in a manner on par with other Langs, it will spread its wings.

https://github.com/apple/swift/blob/master/docs/WindowsBuild.md#MSVC

I'm going to play around with this.. We can also cross compile using WSL -- however, that is slow as all get-out.

**Comment by jhoughjr** [comment]
I wonder if the Swift for TensorFlow windows build and how it works could be useful. I haven't got it fully working yet but it seems to support swift 5.2

**Comment by jefflafay** [comment]
I thought I would share a few resources in case anyone is interested in them.

[This article](https://www.theregister.co.uk/2020/03/30/official_swift_programming_for_windows/) (which funny enough links back to this very github issue) is claiming that swift on windows will be available in Swift 5.3.

It also mentions [swift-build](https://github.com/compnerd/swift-build) as a project that's already creating swift builds for platforms, one of them being Windows 10.

Personally, I've been able to install swift on Ubuntu WSL which appears to be officially supported. So that's another option as well.

**Comment by KyNorthstar** [comment]
I expect Apple will announce official Swift-on-Windows support on Monday. Figure this should be said here

**Comment by d4mn** [comment]
This would be one step forward for all of us!
> 
> 
> I expect Apple will announce official Swift-on-Windows support on Monday. Figure this should be said here

**Comment by DaringDeveloper** [comment]
Apple announces Swift for Windows support is "Coming Soon" and pointed us to swift.org. So swift-build will be supported officially in the near future. Tine to start coding our cross-platform Apps...

I declare SwiftForWindows DEAD!

**Comment by donaldnevermore** [comment]
It's just about time to move on to Swift 5.3+ and contribute to [apple/swift](https://github.com/apple/swift) instead.
